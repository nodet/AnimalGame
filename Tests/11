No
salmon
Does it live in water?
No
No
dog
Does it bark?
Yes
No
dolphin
Is it a mammal?
No
No
Yes
No
Yes
Yes
Yes
No
Yes
Yes
Yes
Yes
Quit
