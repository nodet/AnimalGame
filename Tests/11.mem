cat
No
salmon
Does it live in water?
No
No
dog
Does it bark?
Yes
No
dolphin
Is it a mammal?
