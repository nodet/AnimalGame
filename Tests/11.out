Hi, let's play the animal game. You can quit at any time by typing "Quit."

Think of an animal.
Is your animal a cat? Oh no. What was it? What is a yes/no question to tell a salmon from a cat? Thanks! Let's play again.
Think of an animal.
Does it live in water? Is your animal a cat? Oh no. What was it? What is a yes/no question to tell a dog from a cat? Thanks! Let's play again.
Think of an animal.
Does it live in water? Is your animal a salmon? Oh no. What was it? What is a yes/no question to tell a dolphin from a salmon? Thanks! Let's play again.
Think of an animal.
Does it live in water? Does it bark? Is your animal a cat? Ha! I win! Let's play again
Think of an animal.
Does it live in water? Does it bark? Is your animal a dog? Ha! I win! Let's play again
Think of an animal.
Does it live in water? Is it a mammal? Is your animal a salmon? Ha! I win! Let's play again
Think of an animal.
Does it live in water? Is it a mammal? Is your animal a dolphin? Ha! I win! Let's play again
Think of an animal.
Does it live in water? Ok. Bye!
cat
No
salmon
Does it live in water?
No
No
dog
Does it bark?
Yes
No
dolphin
Is it a mammal?
